.github/ISSUE_TEMPLATE/onboarding-checklist.md

```diff
@@ -52,20 +47,40 @@ Feel free to ask anyone in the group questions about these steps, especially in
     -   [ ] Select and enroll in a retirement plan. You need to enroll in retirment within 30 days of your start date.
     -   [ ] Once you have selected your retirement plan, you can sign up for other benefits. Enroll in other benefits within 31 days of starting.
 
-### Internet
+### Equipment setup
+
+#### Internet
 
 You should be on the university's VPN any time you connect to internet outside the UA.
 
 -   [ ] Install Cisco Secure Client; instructions under [Virtual Private Network (VPN)](https://it.arizona.edu/remote-work).
     -   When asked for a third password/NetID+ method, input "push" or "sms" or "passcode" to autheticate via 2FA.
```

Comment:
Should this line be under the "Log in ..." checkbox below? I don't think you can use this until you're trying to log in